xmodule/capa/safe_exec/tests/test_safe_exec.py

```diff
@@ -86,7 +87,7 @@ def test_cant_do_something_forbidden(self):
         throws SafeExecException, protecting the calling process.
         '''
         # Can't test for forbiddenness if CodeJail isn't configured for python.
-        if not jail_code.is_configured("python"):
+        if not jail_code.is_configured("python") or not is_codejail_rest_service_enabled():
```

Comment:
We could add this as a separate condition, with an appropriate comment about when this is happening and how to configure this in Tutor. It would also be nice to add a reason for skipping this test to `pytest.skip()`.